docs/system_availability_example.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
This is a nice addition. Bringing in analytics around inverter outages versus communication outages is useful for energy loss accounting but not so useful for degradation analysis. Do you envision `rdtools` scope moving towards energy loss accounting?

I'm glad to see more use of energy tags and a flexible 'expected power' tag.

How does the algorithm handle partial day comms or inverter outages? How about overlapping outages?

Reply:
RE the scope, the RdTools tag line has been updated to "RdTools is an open-source library to support reproducible technical analysis of time series data from photovoltaic energy systems."

>How does the algorithm handle partial day comms or inverter outages? How about overlapping outages?

The new availability analysis uses two methods.  The first goes timestamp-by-timestamp to compare inverter power data with system meter power data to distinguish inverter communication/data outages from real inverter power outages.  Because it goes timestamp-by-timestamp, it will detect and classify outages down to the order of the time resolution of the data you give it.  That method can handle multiple inverters offline at a time; it only requires one reporting inverter and the system meter.  The second method is used when the entire system looks to be offline (no inverters communicating production) and works on each outage in its entirety (in contrast to the first method that examines each timestamp in isolation).